README.md

```diff
@@ -89,3 +89,17 @@ Below is an example of regexp patterns used for autolinking, modified in the `co
     }
 },
 ```
+
+## Examples
+
+1. Autolinking `Ticket ####:text with alphanumberic characters and spaces` to a ticket link. Use:
+  - Pattern: `(?i)(ticket )(?P<ticket_id>)( : )(?P<ticket_info>.*)`
+  - Template: `(ticket ${ticket_id}:${ticket_info})`
```

Comment:
If you want to create a link, you have to point to a url, something like `[Ticket ${ticket_id}: ${ticket_info}](https://github.com/mattermost/mattermost-server/issues/${ticket_id})`